de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/fskdbview/FSKDBViewNodeModel.java

```diff
@@ -185,9 +185,41 @@ public void saveCurrentValue(NodeSettingsWO content) {
   @Override
   protected PortObject[] performExecute(PortObject[] inObjects, ExecutionContext exec)
       throws Exception {
+    PortObject outport = inObjects[0];
```

Comment:
This is more of a personal preference, but in the former editor and joiner the input port was also reused as output port and it got complicated when the node was finished and the execute method grew big. I would suggest here to explicitely define an input and output ports so that later when this method grows it will be still be clear.